### PR TITLE
Enhancements to the documentation

### PR DESCRIPTION
**Pull request by mew2057**
Working on condensing and consolidating docs. Improving cross referencing and reducing complexity where possible.

### CODE REVIEW

File: docs/source/cast-big-data/common-problems.rst [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+Timestamps
+----------
+
+Timestamps are generally the number one source of problems in the ELK Stack. This is due to
+a wide variety of precisions and timestamp formats that may come from different data sources.
+
+Elasticsearch will try its best to parse dates, as outlined in the `ELK Date`_ documentation.
+If a date doesn't match the default formats (a usual culprit is epoch time or microseconds) 
+the adminstrator will need to take action.
```
Comment (lasch) [inline docs/source/cast-big-data/common-problems.rst]: s/adminstrator/administrator/

File: docs/source/cast-big-data/common-problems.rst [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    The default ISO 8601 format of Postgresql has precision up to microseconds, requiring
+    postgres generated timestamps to use a parsing strategy.
+
+.. note:: If any indices have been populated with data not interpreted as dates, those 
+    indices will need to be reindexed.
+
+Fixing Timestamps in Elasticsearch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is the prefered methodology for resolving issues in the timestamp. CAST supplies
```
Comment (lasch) [inline docs/source/cast-big-data/common-problems.rst]: s/prefered/preferred/

File: docs/source/cast-big-data/common-problems.rst [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+or any other invalid data types on a per index or index pattern basis:
+
+1. Create a `json` file to store the mapping. CAST recommends naming the file `<template-name>.json` 
+
+2. Populate the file with configuration settings.
+
+    .. code-block:: javascript
+
+        {
+            "index_patterns": ["<NEW INDEX PATERN>"],
```
Comment (lasch) [inline docs/source/cast-big-data/common-problems.rst]: s/PATERN/PATTERN/

File: docs/source/cast-big-data/common-problems.rst [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+The above documentation is a brief primer on how to modify templates, a powerful elasticsearch utility.
+If the user needs more information please consult the official `elastic template documentation`_.
+
+.. _elastic template documentation: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-templates.html
+
+
+Fixing Timestamps in Logstash
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the elasticsearch methodology doesn't apply to the use case, logstash timestamp manipultation
```
Comment (lasch) [inline docs/source/cast-big-data/common-problems.rst]: s/manipultation/manipulation/

File: docs/source/cast-big-data/elasticsearch.rst [hunk 5]
```diff
@@ -32,19 +32,19 @@ in `/opt/ibm/csm/bigdata/elasticsearch/`.
...
 
     :jvm.options: jvm options for the Elasticsearch service.
     :elasticsearch.yml: Configuration of the service specific attributes, please see 
         `elasticsearch.yml`_ for details.
 
-3. Make an ext4 filesystem on each hard drive designated to be in the elastic search JBOD. 
+3. Make an ext4 filesystem on each hard drive designated to be in the Elasticsearch JBOD. 
 
     The mounted names for these file systems should match the names spcified in `path.data`. 
-    Additionally, these mounted file systems should be owned by the `elasticsearch` user and in the
```
Comment (lasch) [inline docs/source/cast-big-data/elasticsearch.rst]: s/spcified/specified/

File: docs/source/cast-big-data/elasticsearch.rst [hunk 6]
```diff
@@ -68,7 +68,7 @@ be data being written to your index.
 Tuning Elasticsearch
 --------------------
 
-The process of tuning and configuring Elasticsearch is incredibly dependen on the volume and
+The process of tuning and configuring Elasticsearch is incredibly dependendent on the volume and
```
Comment (lasch) [inline docs/source/cast-big-data/elasticsearch.rst]: s/dependendent/dependent/

File: docs/source/csm-apis/configuration/configuring-csm-api-logging-levels.rst [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+            "csmd"                      :   "info",
+            "csmras"                    :   "info",
+            "csmapi"                    :   "debug",
+            "csmenv"                    :   "info"
+        }
+    }
+
+
+If you have trouble finding the config files, then daemon config files are located:
+- source repo: "bluecoral/csmconf/”
```
Comment (lasch) [inline docs/source/csm-apis/configuration/configuring-csm-api-logging-levels.rst]: We still have "bluecoral" in there?

**Comment by mew2057** [comment]
@lasch thanks, fixing now.

**Pull request merged**